index.html

```diff
@@ -95,7 +95,31 @@ <h3 class="panel-title">
                             </h3>
                         </div>
                         <div class="list-group scrollable-list">
-                             <a class="list-group-item">
+                            <a class="list-group-item">
+                                <h4 class="list-group-item-heading">Thursday, April 6, 2017</h4>

```

Comment:
Do we want to update this to reflect the reality on the ground that it wasn't available until yesterday?